website/docs/d/advanced_cluster.html.markdown

```diff
@@ -55,7 +55,7 @@ In addition to all arguments above, the following attributes are exported:
 * `id` - The cluster ID.
 * `bi_connector_config` - Configuration settings applied to BI Connector for Atlas on this cluster. See [below](#bi_connector_config). **NOTE** Prior version of provider had parameter as `bi_connector`
 * `cluster_type` - Type of the cluster that you want to create.
-* `disk_size_gb` - Capacity, in gigabytes, of the host's root volume. 
+* `disk_size_gb` - **(DEPRECATED)** Capacity, in gigabytes, of the host's root volume. 
```

Comment:
Should we mention here "use the one under `replication_spec`"?

Reply:
added, lmk WYT